accounts/views.py

```diff
@@ -38,6 +37,11 @@ def role_signup(request, role_form_class, role):
     return render(request, 'accounts/role_signup.html', context)
 
 
+def signup(request):
+    context = []
+    return render(request, "accounts/signup.html", context)
```

Comment:
1. context should be a dict
2. since context is empty, you don't even need to pass it. just write `return render(request, "accounts/singup.html")`